bodhi/server/models.py

```diff
@@ -1967,13 +1967,7 @@ def greenwave_subject(self):
             list: A list of dictionaries that are appropriate to be passed to the Greenwave API
                 subject field for a decision about this Update.
         """
-        # See discussion on https://pagure.io/greenwave/issue/34 for why we use these subjects.
-        subject = [{'item': build.nvr, 'type': 'koji_build'} for build in self.builds]
-        # Additionally add some subject entries for "CI"
-        # https://pagure.io/greenwave/issue/61
-        subject.extend([{'original_spec_nvr': build.nvr} for build in self.builds])
-        subject.append({'item': self.alias, 'type': 'bodhi_update'})
-        return subject
+        return [{'item': self.alias, 'type': 'bodhi_update'}]
```

Comment:
@ralphbean We had talked on Friday about adjusting Bodhi to use just the bodhi_update API. Is this what you had in mind? If so, can you confirm that the ```original_spec_nvr``` bit is not needed? The issue that had been cited via comment is not closed, so I'm curious if removing that is OK at this time.